src/components/CatalogueCoreVariablePanel.vue

```diff
@@ -70,7 +70,11 @@
     name: 'CatalogueCoreVariablePanel',
     methods: {
       getHarmonizationValues (harmonizations) {
-        return harmonizations.map(harmonization => harmonization.sourceLabel).join(', ')
+        if (harmonizations.length > 0) {
+          return harmonizations.map(harmonization => harmonization.id.substring(0, harmonization.id.indexOf('_'))).join(', ')
+        } else {
+          return '-'
```

Comment:
No, I will fix this.

Reply:
This is hard to fix because you do not have the data present. You will need another call to the backend to pick it up and enrich the target harmonizations. I will make an issue of the noted bug and I would suggest to do this in the next release.